panic: cannot write root filesystem: cannot read file .: read .: is a directory

When passing a directory as the first arguments, it throws an error:
```
panic: cannot write root filesystem: cannot read file .: read .: is a directory

goroutine 1 [running]:
github.com/nanovms/ops/cmd.loadCommandHandler(0xc000b06580, 0xc000b0c690, 0x1, 0x3)
        /Users/eyberg/go/src/github.com/nanovms/ops/cmd/cmd_pkg.go:328 +0x597
github.com/spf13/cobra.(*Command).execute(0xc000b06580, 0xc000b0c660, 0x3, 0x3, 0xc000b06580, 0xc000b0c660)
        /Users/eyberg/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:846 +0x2c2
github.com/spf13/cobra.(*Command).ExecuteC(0xc0000d9080, 0x0, 0xffffffff, 0xc000110058)
        /Users/eyberg/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:950 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
        /Users/eyberg/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:887
main.main()
        /Users/eyberg/go/src/github.com/nanovms/ops/ops.go:8 +0x2a
```

--- a/lepton/image.go
+++ b/lepton/image.go
@@ -215,8 +215,13 @@ func BuildPackageManifest(packagepath string, c *types.Config) (*fs.Manifest, er
 	}
 
 	if len(c.Args) > 1 {
-		if _, err := os.Stat(c.Args[1]); err == nil {
-			err = m.AddFile(c.Args[1], c.Args[1])
+		if f, err := os.Stat(c.Args[1]); err == nil {
+			if f.IsDir() {
+				err = m.AddDirectory(c.Args[1], c.Args[1])
+			} else {
+				err = m.AddFile(c.Args[1], c.Args[1])
+			}
+
 			if err != nil {
 				return nil, err
 			}

